library/src/scripts/components/PopoverController.tsx

```diff
@@ -94,15 +111,23 @@ export default class PopoverController extends React.PureComponent<
                 >
                     {this.props.buttonContents}
                 </Button>
+
                 {!this.props.disabled &&
-                    this.props.children({
-                        id: this.contentID,
-                        initialFocusRef: this.initalFocusRef,
-                        isVisible: this.state.isVisible,
-                        closeMenuHandler: this.closeMenuHandler,
-                        renderAbove: this.props.renderAbove,
-                        renderLeft: this.props.renderLeft,
-                    })}
+                    this.state.isVisible && (
+                        <React.Fragment>
+                            {!this.props.openAsModal && this.props.children(childrenData)}
```

Comment:
Instead of doing the check twice here could we use a ternary? `openAsModel ? <Modal /> : this.props.children()`.